src/utils/mlx_list.c

```diff
@@ -138,44 +186,6 @@ mlx_list_t* mlx_lstremove(mlx_list_t** lst, void* value, bool (*comp)(void*, voi
 	return (lstcpy);
 }
 
-// Retrieve Z value from queue.
-static int32_t mlx_getdata(mlx_list_t* entry)
```

Comment:
I fail to see the reason for this change, maybe having statics at the front could be nice but here its just because the function that uses it is just below it. So its imo more convenient to keep it here.